test/test_docker_upload.py

```diff
@@ -14,10 +14,16 @@ def setUpClass(cls):
         cls.test_environment = exaslct_utils.ExaslctTestEnvironmentWithCleanUp(
             cls, exaslct_utils.EXASLCT_DEFAULT_BIN
         )
+        cls.itde_test_environment = exaslct_utils.ExaslctTestEnvironmentWithCleanUp(
```

Comment:
In theory, we could also replace it by the API, because the itde cli is not what we test

Reply:
But it's easier to use the ExaslctTestEnvironmentWithCleanUp() as it prepares the parameter for `spawn_test_environment()` with the common values.

Reply:
I meant the API Version of the Environment

Reply:
Done